internal/server/proto/server.proto

```diff
@@ -491,8 +491,12 @@ message Variable {
     int64 num = 10;
     // hcl stores values of any complex type in a raw string format, and
     // converts it to hcl when we evaluate variables. This is used when
-    // loading values from a file or from the server/UI.
+    // loading simple typed values from a file or simple/complex types 
+    // from the server/UI.
     string hcl = 3;
+    // complexhcl stores any complex typed hcl value provided via a values file
+    // in bytes, converting it back to its hcl expression when we evaluate variables.
+    bytes complexhcl = 11;
```

Comment:
Why do we need two separate types for this? I'm looking at the PR but not seeing the fundamental difference...

Reply:
Because I am tired and my brain skipped right over making a buffer. 😔

Reply:
Update incoming